### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by jamesjer** [libsemigroups/HPCombi#36]
I am looking at packaging HPCombi for Fedora, and would like to be sure I understand the license.  The top-level LICENSE file contains the text of GPL version 3.  However, nowhere is the actual version of the GPL specified.  Many files have this text at the top:
```
//  Distributed under the terms of the GNU General Public License (GPL)       //
//                                                                            //
//    This code is distributed in the hope that it will be useful,            //
//    but WITHOUT ANY WARRANTY; without even the implied warranty of          //
//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU       //
//   General Public License for more details.                                 //
//                                                                            //
//  The full text of the GPL is available at:                                 //
//                                                                            //
//                  http://www.gnu.org/licenses/                              //
```
That URL provides access to every version of the GPL.  The Fedora licensing guidelines state that in this case, we presume the license to be GPL-1.0-or-later; i.e., that those who receive the code can choose to comply with *any* version of the GPL, including version 1.0.  Is this your intention?

If you intend to specify GPL version 3, do you mean GPL-3.0-only or GPL-3.0-or-later?

It would be great if the exact license, with version number and "only" or "any later version", were specified somewhere in the distribution.

**Comment by james-d-mitchell** [comment]
Thanks for the question @jamesjer, @hivert what do you think? I'm guessing we want GPL-3.0-or-later?

**Comment by hivert** [comment]
@jamesjer @james-d-mitchell GPL-3.0-or-later is good for me indeed.

**Comment by james-d-mitchell** [comment]
Thanks @hivert i'll make another release once the license statements are up to date

**Comment by jamesjer** [comment]
Great, thank you for the quick action.

**Comment by hivert** [comment]
I just sedded the file to fix the copyright statement. It should be Ok now. @james-d-mitchell : may I leave it to you to make the release ? Thanks !

**Comment by james-d-mitchell** [comment]
Hopefully resolved in v1.0.1, I'd be most grateful if you'd confirm this @jamesjer

**Comment by jamesjer** [comment]
Version 1.0.1 looks great.  Thank you so much for taking care of this so quickly.  HPCombi should be in Fedora 40, which will be released in April.
